### PR TITLE
Revise the modified parameters of the Bit components in the TodoTemplate (#1931)

### LINKED ISSUE #1931: Correct the old parameters of the Bit components used in the `TodoTemplate` project based on the new updates
Some of the Bit components have been updated and some parameters have been renamed that have not been updated in the `TodoTemplate` project.
For example, the `BitRadioButtonGroup` component parameter `Key` should be changed to the `Value`

### PR DESCRIPTION
**Pull request by MohammadJavadTabari**
this closes #1931

### CODE REVIEW

File: src/Tooling/Bit.Tooling.Templates/TodoTemplate/Web/Pages/EditProfile.razor [hunk 1]
```diff
@@ -71,9 +71,9 @@
                 <BitLabel>Gender</BitLabel>
                 <BitRadioButtonGroup @bind-SelectedKey="UserToEdit.GenderAsString"
```
Comment (Cyrus-Sushiant) [inline src/Tooling/Bit.Tooling.Templates/TodoTemplate/Web/Pages/EditProfile.razor:72]: Plz, use `@bind-Value` instead of `@bind-SelectedKey`

**Review by Cyrus-Sushiant — CHANGES_REQUESTED** [review]

**Comment by msynk** [comment]
you should not put anything other than the `this closes ...` in the description of the PR. if you need to add some info to your PR plz use the comment.

**Comment by msynk** [comment]
@Cyrus-Sushiant 
are we sure that there aren't any more required updates?

**Comment by msynk** [comment]
@MohammadJavadTabari 
plz resolve the conflict

**Comment by Cyrus-Sushiant** [comment]
> @Cyrus-Sushiant are we sure that there aren't any more required updates?

No. The `DefaultIsChecked` parameter of BitCheckbox in `Todo.razor` page must be changed

**Comment by MohammadJavadTabari** [comment]
@msynk 
@Cyrus-Sushiant 
thank's for your comments, I will fix it now

**Comment by MohammadJavadTabari** [comment]
@msynk
@Cyrus-Sushiant 
what mentioned above is done, and I keep searching for more problems like that.
plz, tell me if you see more unsolved problems related to this issue.

File: src/Tooling/Bit.Tooling.Templates/TodoTemplate/Web/Pages/EditProfile.razor (hunk 1 above)
Comment (Cyrus-Sushiant) [reply src/Tooling/Bit.Tooling.Templates/TodoTemplate/Web/Pages/EditProfile.razor]: Please fix the above issue. @MohammadJavadTabari

**Review by maryamhdr — APPROVED** [review]

**Review by Cyrus-Sushiant — APPROVED** [review]

**Review by msynk — APPROVED** [review]

**Pull request merged**